### DISCUSSION TITLE
Location Check Not Working

### DISCUSSION THREAD

**Discussion by asamohammed** [community/community#42545] [GitHub Education]
<img width="1388" alt="errorMessage" src="https://user-images.githubusercontent.com/106772665/209422207-3db043b2-7d97-463c-8f9d-986159b4daa0.png">

Hi,

This error message keeps popping up and I don't know how to fix it, I am on the school campus.
I've been trying to fix this for 3 weeks now. I've tried different locations, wifi, computer, phones, even went to the school IT dept.

I am connected to the school wifi and am in the middle of campus. 
Does the system check your GPS or your IP address for location?

**Comment by samus-aran** [comment]
Hi @asamohammed 👋 

How did you get on?

**Reply by asamohammed** [reply]
@samus-aran I tried for months and nothing worked. Then I visited a friend for the holidays and tried submitting it and it went through. I was over 300 miles away from my school. 
Could have been a coincidence or GitHub just disabled that location check.
